nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -214,6 +214,11 @@ language governing permissions and limitations under the License. -->
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-standard-prioritizers</artifactId>
```

Comment:
The version for this artifact is specified in the nifi-framework-bundle, it should probably be moved up to the top-level POM's dependencyManagement section since now another NAR depends on it. I couldn't build this PR from scratch without specifying the version here.